test-app/src/androidTest/java/com/linkedin/parser/test/junit4/java/BasicJUnit4.java

```diff
@@ -12,13 +12,13 @@
 public class BasicJUnit4 extends ConcreteTest {
 
     @Test
-    @TestValueAnnotation(stringValue = "On a method", intValue = 12345, boolValue = true, longValue = 56789L)
+    @TestValueAnnotation(stringValue = "On a method", intValue = 12345, boolValue = true, longValue = 56789L, enumValue = TestEnum.SUCCESS)
     public void basicJUnit4() {
         assertTrue(true);
     }
 
     @Test
-    @TestValueAnnotation(floatValue = 0.25f, doubleValue = 0.5, byteValue = 0x0f, charValue = '?', shortValue = 3)
+    @TestValueAnnotation(floatValue = 0.25f, doubleValue = 0.5, byteValue = 0x0f, charValue = '?', shortValue = 3, enumValue = TestEnum.FAIL)
```

Comment:
nitpick: I'd keep current pattern that one type only shows up once, either in basicJUnit4() or basicJUnit4Second()